sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/_simulator.py

```diff
@@ -24,6 +24,8 @@
 from ._helpers import ConversationHistory, Turn
 from ._utils import JsonLineChatProtocol
 
+if len(USER_AGENT) < 24:
+    USER_AGENT += "/simulator"
```

Comment:
Do you have an example how the final USER_AGENT would look like? Below is the UA format. I think we should design this thing a littlel bit so we can track not only the simulator usage but also the simulator usage per type. For example: 

`azure-ai-evaluation/1.0.0 (type=Evaluator; subtype=FluencyEvaluator)`

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent
![image](https://github.com/user-attachments/assets/5efa5a96-f54a-4420-a27e-90cd3f9b30c9)

Reply:
good idea, I'll make the change. Not sure if there's a type of simulator in non-adversarial ones, but I'll make the change to follow the format

Reply:
Shall we use ClassName as the subtype? So `azure-ai-evaluation/1.1.0 (type=simulator subtype=Simulator)
I want to use same pattern for evaluators as well.

Reply:
Feels like duplicate data, but I can do that if you want. 

Not sure how those long user agents would work because of the 24 char limit:
`len("azure-ai-evaluation/1.1.0") == 25`
`len("azure-ai-evaluation/1.1.0 (type=simulator)") == 42`
`len("azure-ai-evaluation/1.1.0 (type=simulator subtype=Simulator)") == 60`

Reply:
changed